src/lib.rs

```diff
@@ -78,7 +78,7 @@ extern crate rcgen;
 use rcgen::generate_simple_self_signed;
 # fn main () {
 let subject_alt_names :&[_] = &["hello.world.example".to_string(),
-	"localhost".to_string()];
+    "localhost".to_string()];
```

Comment:
Why does this whitespace change exist? What does clippy complain about it?

Reply:
See here: https://rust-lang.github.io/rust-clippy/master/index.html#tabs_in_doc_comments

Reply:
Thanks. rcgen uses tabs for indentation so the lint is wrong. Please don't do it.